packages/terra-show-hide/tests/jest/ShowHideFocuser.test.jsx

```diff
@@ -12,8 +11,14 @@ describe('ShowHideFocuser', () => {
     return <p><ShowHideFocuser ref={ref} prefix={prefix} focusableText={focusableText} /></p>;
   };
 
-  afterAll(() => {
-    jest.resetAllMocks();
+  let mockSpyUuid;
+
+  beforeEach(() => {
+    mockSpyUuid = jest.spyOn(uuidv4, 'v4').mockImplementation(() => '00000000-0000-0000-0000-000000000000');
+  });
+
+  afterEach(() => {
+    mockSpyUuid.mockRestore();
```

Comment:
This should be beforeAll and afterall respectively since we need to do these once.

Reply:
@sdadn fixed in https://github.com/cerner/terra-core/pull/3894/commits/cf1e86816385f9145c141575762fe555ad8c19f8. Let me know if this approach is good enough to be implemented terra-wide